corehq/apps/hqwebapp/views.py

```diff
@@ -1276,3 +1280,49 @@ def temporary_google_verify(request):
     # will remove once google search console verify process completes
     # BMB 4/20/18
     return render(request, "google9633af922b8b0064.html")
+
+
+@require_POST
+@csrf_exempt
+def log_email_event(request):
```

Comment:
Looks like this allows unauthenticated access? That seems like a security concern. Is there a way to have SES use an API key or something so we can authenticate these requests?

Reply:
I think the only way we could authenticate it is by encoding something in the URL with a shared secret (i.e. `/log_email_event/SECRET`) as we do with inbound SMS. 

Although I'm curious what the attack vector could be here? No data flows back out of the system. I suppose some malicious actor could change messages to be marked as delivered, even if they weren't (but then they'd need to know the specific ID of the messaging subevent... which we do send in plaintext in the email header, so it's possible).

I'm going to update this with the secret anyway, as I think it is a small lift.

Reply:
It just seemed like an unnecessary vulnerability to allow anyone who knows a message subevent ID to change the status of that event by hitting a URL on our system.